whisperv6/events.go

```diff
@@ -13,6 +13,8 @@ const (
 	EventEnvelopeSent EventType = "envelope.sent"
 	// EventEnvelopeExpired fires when envelop expired
 	EventEnvelopeExpired EventType = "envelope.expired"
+	// EventBatchAcknowledged is sent when envelope was acknowleged by a peer
+	EventBatchAcknowledged EventType = "batch.acknowleged"
```

Comment:
The description says `is sent when **envelope** was acknowleged` but the event is called `EventBatchAcknowledged`.